frontend/src/modules/forum/forum-iframe.jsx

```diff
@@ -58,16 +61,10 @@ const ForumIframe = ({
       console.log(e.data)
 
       if (e.data === 'loaded') {
-        const isValidUser =
-          profile?.reviewStatus === 'APPROVED' && profile?.chatAccountId
-
-        if (!isLoggedInRocketChat && isValidUser) {
+        if (profile?.chatAccountId) {
```

Comment:
Hi @martinchristov , from my understanding, the chatAccountId indicates the user has been linked to RC. if the user doesn't have that then the SSO will not work.

Reply:
It's basically what Iwan is saying. Technically the SSO will work but the outcome of it is different. In theory the user will always have it but in case it doesn't we don't try. If you do SSO without a `chatAccountId` it will show the registration form of RC if I'm not mistaken. I would say that maybe this could be an improvement to redirect the user to a place where it can activate the chat account but as I said at the moment it seems you will always have the `chatAccountId`.

Reply:
From what I'm seeing, if you don't have the `chatAccountId` nothing will happen. So it will be the endless spinner again. In that case why are we even making this conditional?

Reply:
Good point. In that case the conditional is for nothing. But if it leads to an endless spinner if the user don't have the `chatAccountId` shouldn't we do something about it? For example, redirecting the user somewhere to activate the account? I mean in the hypothetical case it happens you don't have that `chatAccountId`.

Reply:
My first thought was that the conditional was just to prevent calling SSO. But, I realized it doesn't have an effect. 

For me, handling the user doesn't have chat AccountId makes sense, because of my understanding of the `chatAccountId` not being instantly filled when we successfully registered
![image](https://github.com/akvo/unep-gpml/assets/20871229/d7cfe32c-4425-4e16-9de7-855c35919725)

> I guest this email came from a worker/job, no?

Reply:
Yes. That email though is only on test environment. We don't use it on production. But yes, it confirms that the account was created and activated.

Reply:
I mean, the point is if we leave it there we need to give it some meaning. That is, if we see an user don't have a chat account for some reason we do something. Otherwise just remove it.

Reply:
I see, I think I'd prefer to remove it since this is a high priority.